deploy/nuodb-scc.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  annotations:
+    kubernetes.io/description: An SCC for use by NuoDB Helm chart deployments
+  name: nuodb-scc
+allowHostDirVolumePlugin: true
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: true
+allowPrivilegedContainer: false
+allowedCapabilities:
+  - FOWNER
+defaultAddCapabilities:
+  - FOWNER
+fsGroup:
+  type: MustRunAs
+groups: null
+priority: 100
+readOnlyRootFilesystem: false
+requiredDropCapabilities:
+  - KILL
+  - MKNOD
+  - SETUID
+  - SETGID
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: MustRunAs
+  seLinuxOptions:
+    level: s0
+    type: spc_t
+supplementalGroups:
+  type: RunAsAny
+volumes:
+  - configMap
+  - downwardAPI
+  - emptyDir
+  - hostPath
+  - persistentVolumeClaim
+  - projected
+  - secret
```

Comment:
From the OpenShift docs (https://docs.openshift.com/enterprise/3.0/architecture/additional_concepts/authorization.html#security-context-constraints), it looks like a declarative way to add the SCC to the service account might be something like the following:
```
users: 
- system:serviceaccount:<the namespace>:{{ .Values.nuodb.serviceAccount }}
```

Reply:
This was what was being done but Martin moved it to the doc for this as it's not deployed by the charts (else you could use the namespace passed in values / helm)

Reply:
I do not automatically deploy the SCC. The user has to do it. Because of that, I can't pass the namespace in from HELM, there is no template to fill out.